ztp/gitops-subscriptions/argocd/example/policygentemplates/test-sno.yaml

```diff
@@ -16,14 +16,14 @@ spec:
   mcp: "master"
   sourceFiles:
     - fileName: SriovNetwork.yaml
-      policyName: "sriov-nw-fh-policy"
+      policyName: "config-policy"
```

Comment:
Each file (common, group, test-sno) will need a uniqe policy name, otherwise they will overwrite one another.

Reply:
ah wouldn't the policy names gets prepended by `common` or `test-sno`? 

Do have any suggestion for the name? `config-policy-common`, `config-policy-group` and `config-policy-sno` ?

Reply:
Yes, apologies. The generated policy name will be `<pgtName>-<policyName>`. So `common-config-policy`, `group-du-sno-config-policy` and `test-sno-config-policy`.